tests/backend-storage/package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@itwin/cloud-agnostic-core": "workspace:*",
     "@itwin/object-storage-core": "workspace:*",
+    "abort-controller": "~3.0.0",
```

Comment:
Is it expected that every user would have to find their own implementation of abort-controller to use? If so, why use one that hasn't been published for 3 years and is no longer compliant with whatwg specification for AbortController? It will cause issues when moving to node.js 18.

Reply:
Also, with newer versions of node.js, would this work without importing any additional dependencies?

Reply:
> Is it expected that every user would have to find their own implementation of abort-controller to use?

Yes. 

> If so, why use one that hasn't been published for 3 years and is no longer compliant with whatwg specification for AbortController?

Problem here is that most popular `AbortController` implementations don't comply with whatwg specification ([node-abort-contoller](https://www.npmjs.com/package/node-abort-controller), [abortcontroller-polyfill](https://www.npmjs.com/package/abortcontroller-polyfill)). This is the reason why new `AbortSignal` interface doesn't contain `reason` or `throwIfAborted`. So unless we want to create and expose another `AbortController` implementation, we should use that is easily available for consumers.

> Also, with newer versions of node.js, would this work without importing any additional dependencies?

It would. This works with `AbortController` provided by Node 16.